app/Http/Controllers/CategoryController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace App\Http\Controllers;
+use App\Http\Controllers\Controller;
+use App\Models\Category;
+use Illuminate\Http\Request;
+
+class CategoryController extends Controller
+{
+
+    public function index()
+    {
+        $Categories = Category::all();
+        return view('admin.category', compact('Categories'));
+    }
+
+
+    public function create()
+    {
+        return view('admin/create-category');
+    }
+
+
+    public function store(Request $request)
+    {
+        $request->validate([
+            'title'=>['required', 'string', 'max:255'],
+            'meta_title'=>['required', 'string', 'max:255'],
+        ]);
+
+        $slug = str_replace(' ', '-', $request->meta_title);
+
+        Category::create([
+            'title' => $request->title,
+            'meta_title' => $request->meta_title,
+            'slug' => $slug,
+        ]);
+
+        return redirect()->back()->with('success', 'Category created');
+    }
+
+
+    public function show(Category $category)
+    {
+        return view('/admin/viewcategory', compact('category'));
```

Comment:
Use dot(.) instead of (/) when specifying a blade template. The two will work though. But I prefer the dot sha. Its a matter of choice